### ISSUE TITLE
Ability to decide when to setup routes, before of after provision

### ISSUE THREAD

**Issue by krzysztof-magosa** [vagrant-dotvm/vagrant-dotvm#7]

**Comment by jhcook** [comment]
I'm starting to think that even though there are multiple ways to make this persistent across various Unix/Linux distributions, that we should actually do it that way. There are various ways to do it at the command line (route and ip). Although there are only a couple, making this persistent will remove headaches.

**Comment by jhcook** [comment]
Routes should be added before main provisioning as in many cases the route may be needed to complete provisioning.

**Comment by krzysztof-magosa** [comment]
It's the current behaviour.

Order looks like that:
- configure routes
- configure /etc/hosts
- configure provisions

**Comment by krzysztof-magosa** [comment]
At the moment I don't need ability to move `configure routes` to different place, if it's the same for you we can close this ticket as `won't fix`

**Comment by jhcook** [comment]
It doesn't need to be moved, but needs to be run only once or with --provision. The biggest issue is making it persistent which I'm perfectly fine with doing. As long as we support RHEL and Debian we should be fine.

**Comment by krzysztof-magosa** [comment]
Marking as `won't fix`.
Instead of that network configuration persistance will be implemented in issue #13.
